dask_kubernetes/operator/tests/resources/simplecluster.yaml

```diff
@@ -18,4 +18,5 @@ spec:
     # serviceAccountName: null
   replicas: 3
   resources: {}
-  env: {}
\ No newline at end of file
+  env: {}
+  adaptive: "false"
```

Comment:
Kopf timer handler keeps checking `adaptive`. Once `adaptive` is set true, the operator ensures that the number of workers is between `minimum` and `maximum` and scales up and down depending on the workload. Then we can set `adaptive` to false when manual scaling is used and/or we provide a method for the user to stop adaptive mode.

Reply:
This is one approach. However it does means we would have `replicas`, `minimum`, `maximum` and `adaptive` to control scaling and we effectively would have to modes, adaptive and not adaptive.

I wonder if we could introduce `minimum` and `maximum` as a replacement and effectively just have an always-on adaptive mode. And somehow configure things so that if you scale to a fixed number then `minimum` and `maximum` are just set to the same value. This also has the benefit of failed workers being restarted automatically, something that doesn't happen with static scaling today.